core/src/main/java/org/elasticsearch/search/aggregations/Aggregations.java

```diff
@@ -98,4 +107,36 @@ public final boolean equals(Object obj) {
     public final int hashCode() {
         return Objects.hash(getClass(), aggregations);
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        if (aggregations.isEmpty()) {
+            return builder;
+        }
+        builder.startObject(AGGREGATIONS_FIELD);
+        toXContentInternal(builder, params);
+        return builder.endObject();
+    }
+
+    /**
+     * Directly write all the aggregations without their bounding object. Used by sub-aggregations (non top level aggs)
+     */
+    public XContentBuilder toXContentInternal(XContentBuilder builder, Params params) throws IOException {
+        for (Aggregation aggregation : aggregations) {
+            aggregation.toXContent(builder, params);
+        }
+        return builder;
+    }
+
+    //TODO add tests for this method
```

Comment:
would you mind if I add tests as a follow-up so we can get this in rather quickly?